### ISSUE TITLE
Bad style in Recurrence Editor Dropdowns (Material Theme)

### ISSUE THREAD

**Issue by BeniFreitag** [telerik/kendo-angular#3313]
**Describe the bug**
All Dropdowns at "Repeat on" are almost unreadable, because they're missing a width or padding.

**To Reproduce**
1. Open https://www.telerik.com/kendo-angular-ui/components/scheduler/editing/custom-reactive-editing/
2. Ensure Theme is "Material"
2. DoubleClick any recurring item > Edit the series
3. Change to Monthly/yearly
4. Open any dropdown, all have the wrong width

**Expected behavior**
Dropdowns should have a width, so no horizontal scrollbar is shown.

**Screenshots**
![image](https://user-images.githubusercontent.com/6669093/118990727-6cac0800-b983-11eb-8f86-3411a771aa69.png)

this style was there in earlier versions:
```css
.k-dropdown .k-dropdown-wrap .k-input {
  padding: 0.375rem 0.75rem;
}
```

**Comment by BeniFreitag** [comment]
Dropdowns look fine now, but the labels are in a strange order

![image](https://user-images.githubusercontent.com/6669093/171849112-22d55590-6319-4fb4-abb9-b9009129a2bb.png)

**Comment by stnikolova** [comment]
The form layout was addressed in v13.0.0 of @progress/kendo-angular-scheduler. See [demo](https://www.telerik.com/kendo-angular-ui/components/scheduler/editing/editing-reactive-forms/).
